content/en/guides/core/registry/model_registry_eol.md

```diff
@@ -53,30 +56,76 @@ Registered models are now referred to as *collections*.
 
-### Artifact path continuity
+The following will happen during the migration:
 
-No action is currently required.
+* W&B will migrate registered models (now called *collections*) and associated artifact versions [from the legacy Model Registry to the W&B Registry]({{< relref "#artifact-paths" >}}). 
+* API calls that reference the legacy Model Registry will be [redirected to the new W&B Registry]({{< relref "#legacy-model-registry-apis" >}}). This means that any existing references to model artifacts in your code or workflows will continue to function without requiring updates.
+* Team level visibility of the legacy Model Registry will be [transitioned to organization level visibility]({{< relref "#team-visibility-to-organization-visibility" >}}) in the W&B Registry.
+* The legacy Model Registry will transition to a [read-only state]({{< relref "#legacy-model-registry-read-only-state" >}}) once the migration begins and will remain accessible for reference.
 
-## During the migration
 
-W&B will initiate the migration process. The migration will occur during a time window that minimizes disruption to W&B services. The legacy Model Registry will transition to a read-only state once the migration begins and will remain accessible for reference.
 
 ## After the migration
 
 Post-migration, collections, artifact versions, and associated attributes will be fully accessible within the new W&B Registry. The focus is on ensuring that current workflows remain intact, with ongoing support available to help navigate any changes.
 
+
+### Artifact paths
+
+Legacy model registry path will be redirected to the new W&B Registry path. This means that any existing references to model artifacts in your code or workflows will continue to function without requiring updates.
+
+For example, if you have a model artifact path in your code like:
+
+```python
+model_reg_path = "team-name/model-registry/collection-name:version"
+```
+
+W&B will automatically redirect to its new W&B Registry path:
+
+```python
+registry_path = "org-name/wandb-registry-team-name/collection-name:version"
+```
+
+### Legacy Model Registry APIs
+
+The following API calls will be supporting automatic redirects to the new W&B Registry:
+
+* `wandb.Api().artifact()`
+* `wandb.Api().use_artifact()`
+* `wandb.run.link_artifact()`
+* `wandb.Artifact.link()`
+
+{{% alert title="Python SDK version"%}}
+Based on your W&B Python SDK version, the redirection behavior will vary:
+
+* Users with Python SDK versions v0.22 and greater will receive a non-breaking warning in your SDK logs about this redirection, encouraging users to update paths in new code over time.
+* Users with Python SDK versions below v0.22 will not receive a warning, but the redirection will still occur.
+
+{{% /alert %}}
+
+<!-- Add note about degraded experience? If yes, get more concrete details from the engineering team. -->
```

Comment:
Will defer to @trane293 / @tristan-spaulding2 on whether we should include details. Happy to provide more info once we have that confirmation!